src/functionalTest/java/uk/gov/hmcts/opal/Junit5RunnerTest.java

```diff
@@ -1,12 +1,14 @@
 package uk.gov.hmcts.opal;
 
 import org.junit.platform.suite.api.IncludeEngines;
+import org.junit.platform.suite.api.IncludeTags;
 import org.junit.platform.suite.api.SelectClasspathResource;
 import org.junit.platform.suite.api.Suite;
 
 @Suite
 @IncludeEngines("cucumber")
 @SelectClasspathResource("features")
+@IncludeTags({"PO-311","PO-312", "PO-315"})
```

Comment:
This isn't really necessary and will stop the execution of the other tests, tag the feature files with '@Opal' as well and they will be picked up when the gradle task 'functional' is ran

Reply:
Look at existing feature files for examples